### PR TITLE
Documenting modifying MySQL cnf file and restarting the service.

### PR DESCRIPTION
**Pull request by quicksketch**
Follow-up from https://github.com/Lullabot/tugboat/issues/4666 and https://github.com/TugboatQA/images/issues/85: Document the ability to restart the MySQL service.

### CODE REVIEW

**Review by q0rban — APPROVED** [review]

File: content/reference/tugboat-images.md [hunk 1]
```diff
@@ -82,6 +82,19 @@ services:
         - mysql -e "CREATE DATABASE foo;"
 ```
 
+If MySQL or MariaDB daemon configuration needs to be changed, create or modify the configuration file and
+restart the service ([which is managed using runit](/setting-up-services/how-to-set-up-services/running-a-background-process/)):
```
Comment (q0rban) [inline content/reference/tugboat-images.md:86]: Our linter wants to make this change:

```suggestion
If MySQL or MariaDB daemon configuration needs to be changed, create or modify the configuration file and restart the
service ([which is managed using runit](/setting-up-services/how-to-set-up-services/running-a-background-process/)):
```

**Review by q0rban — CHANGES_REQUESTED** [review]

File: content/reference/tugboat-images.md [hunk 2]
```diff
@@ -82,6 +82,19 @@ services:
...
+
+```
+services:
+  mysql:
+    image: tugboatqa/mysql
+    commands:
+      init:
+        - echo "max_allowed_packet=536870912" >> /etc/mysql/conf.d/tugboat.cnf
+        - sv restart mysql
+```
```
Comment (quicksketch) [inline content/reference/tugboat-images.md:96]:
```suggestion
```
Note the service restarts when the preview is committed, restarting may only be necessary when wanting the configuration
to be applied before other actions, such as importing the database.
```

File: content/reference/tugboat-images.md [hunk 3]
```diff
@@ -82,6 +82,19 @@ services:
...
+
```
Comment (quicksketch) [inline content/reference/tugboat-images.md:97]:
```suggestion

Note the service restarts when the preview is committed, restarting may only be necessary when wanting the configuration
to be applied before other actions, such as importing the database.

```

**Comment by quicksketch** [comment]
Thanks! Linter changed made and I added a sentence about how restarting is only necessary if it affects the import.

**Review by q0rban — APPROVED** [review]

**Pull request merged**